### PR TITLE
[Form] Support inverted transparent textarea

### PR DESCRIPTION
**Pull request by lubber-de**
## Description

Support for transparent textarea in an inverted segment or form (only the color needed to be changed)

### Related to
#158

### CODE REVIEW

File: src/definitions/collections/form.less [hunk 1]
```diff
@@ -1117,4 +1117,13 @@
   font-size: @massive;
 }
 
+
+.ui.inverted.form textarea.transparent,
+.ui.form .inverted.segment textarea.transparent {
+  color: inherit;
+}
+
```
Comment (hammy2899) [inline src/definitions/collections/form.less]: Why so much whitespace?

Comment (lubber-de) [reply src/definitions/collections/form.less]: I simply adjusted the logic the same as it is for input.

... or maybe i don't get, what you mean? :thinking:

Comment (lubber-de) [reply src/definitions/collections/form.less]: if you simply mean the additional empty lines...then i am simply sorry for that, there was no reason for sure

Comment (ColinFrick) [reply src/definitions/collections/form.less]: What he probably meant is that you added two empty lines before the selector and four lines after.
Additionally couldn't this be added to this:
https://github.com/fomantic/Fomantic-UI/blob/31ed0007b7133967b7ef2ad2d52cc8328acf4299/src/definitions/collections/form.less#L162-L173
or to the inverted variation section?
https://github.com/fomantic/Fomantic-UI/blob/31ed0007b7133967b7ef2ad2d52cc8328acf4299/src/definitions/collections/form.less#L714-L749

Comment (hammy2899) [reply src/definitions/collections/form.less]: You have 4 blank lines of whitespace added and there isn't any need for it

**Comment by hammy2899** [comment]
I agree with @ColinFrick you can probably achieve this without making another textarea definition.

**Comment by lubber-de** [comment]
@ColinFrick Thanks, i moved it to a more suitable area, taking care of no extra whitespace now, within the less file as suggested. I'll take care of such things in the future! :blush:

File: src/definitions/collections/form.less (hunk 1 above)
Comment (lubber-de) [reply src/definitions/collections/form.less]: ok, got it now working in the suggested transparent section also. see latest commit. Sorry for any inconvenience :wink:

**Pull request merged**